charts/kube-plex/templates/deployment.yaml

```diff
@@ -34,6 +34,9 @@ spec:
         {{- if .Values.kubePlex.loglevel }}
         kube-plex/loglevel: "{{ .Values.kubePlex.loglevel }}"
         {{- end }}
+        {{- if .Values.kubePlex.eaeRootDir }}
+        kube-plex/eae-root-dir: "{{ .Values.kubePlex.eaeRootDir }}"
```

Comment:
I think this is missing the corresponding code in metadata.go (around line 100 or so), which parses the value from the pod definition.

I can take a look and add it a bit later in the week, but wanted to give you a heads up :D

Reply:
Thanks! What I added in 61b1164 hopefully does the trick.